ee/clickhouse/sql/elements.py

```diff
@@ -71,7 +73,8 @@
 
 INSERT_ELEMENTS_SQL = """
 INSERT INTO elements SELECT 
-    generateUUIDv4(), 
+    %(uuid)s,
+    %(event_uuid)s, 
```

Comment:
Hmm... I think the point of elements was that they exist independently from events? Is there a reason for putting the ID of the first event that creates the element in an event?

Reply:
for traceability more than anything

Reply:
Fair enough I guess.
Though I'd personally name it `from_event_uuid` or something like that. Otherwise it looks like a proper 1:1 mapping.